Swerve/SwerveModules/SwerveModule.java

```diff
@@ -142,4 +166,23 @@ public LinearVelocity accelLimit(LinearVelocity newVelocity) {
 
         return newVelocity;
     }
+
+
+    @Override
+    public void updateInputs(ModuleIOInputsAutoLogged inputs) {
+        // Yet to be implemented
+        
+
+        
+    }
+
+    public void periodic() {
+        updateInputs(inputs);
+        Logger.processInputs("Drive/Module" + Integer.toString(swerveModuleConstants.moduleNumber), inputs);
+
+        // Update alerts
+        driveDisconnectedAlert.set(!inputs.driveConnected);
```

Comment:
No need to add errors for disconnected motors, CAN errors are already populated in the console

Reply:
Could the CAN error be changed so that it doesn't appear when implementing a SimIO class for the module?

Reply:
I dont think CAN errors populate for simulation devices?